contrib/pipelines/secadvisor/README.md

```diff
@@ -5,30 +5,104 @@ Implements the IBM Security Advisor pipeline
 
 ## Setup Skydive
 
-First build and run skydive:
+Follow instructions on http://skydive.network/documentation/build to install prerequisites and prepare your machine to build the skydive code.
+Build and run skydive:
 
 ```
 make
 cp etc/skydive.yml.default skydive.yml
-sudo $(which skydive)) allineone -c skydive.yml
+// adjust settings in skydive.yml
+sudo $(which skydive) allineone -c skydive.yml
 ```
 
+Alternatively, run a skydive agent on each of your hosts and a skydive analyzer on one host.
+
+On each machine:
+
+```
+sudo skydive agent -c skydive.yml
+```
+
+Be sure to set the field "analyzers" in the skydove.yml to point to the analyzer.
+
+On one machine:
+
+```
+sudo skydive analyzer -c skydive.yml
+```
+
+
 ## Setup Minio
 
-Install and run minio (an AWS S3-like object store).
+Follow the instructions on https://docs.min.io/docs/minio-quickstart-guide.html to install and run a minio object store (an AWS S3-like object store).
+
+```
+wget https://dl.min.io/server/minio/release/linux-amd64/minio
+chmod +x minio
+./minio server /data
+```
+
+This will run a basic minio object store server on your machine, placing the objects in "/data".
+Note the printed output in the logs from the ./minio command and take down the values of "Endpoint", "AccessKey", and "SecretKey". Place these items in the secadvisor.yml in the appropriate locations (see below). 
+Create a bucket in the object store to hold the security-advisor data.
+
+## Prepare secadvisor.yml
+
+```
+cp secadvisor.yml.default secadvisor.yml
+```
+
+Set the fields "endpoint", "access_key", "secret_key", "bucket", "cluster_net_masks", etc.
+The subnets specified in cluster_net_masks are used to determine whether a flow is internal, ingress, or egress. Enter in the list all of the subnets that are recognized as being inside your cluster.
+The types of "excluded_tags" that are recognized are: internal, ingress, egress, other.
 
 ## Setup Pipeline
 
-Build and run the pipeline:
+Build and run the pipeline in the secadvisor directory:
 
 ```
 make all
 make install
 make run
 ```
 
+or
+
+```
+secadvisor secadvisor.yml
```

Comment:
@KalmanMeth This is how to activate the pipeline, put this in a new section ? in this case we need to explain that someone can start several of those instances in prallel if they want to write to more than one bucket/destination (and in general more than one pipeline).

in the future there will be filtering capabilities based on code from @masco-enovance  that allow to define a set of 'capture-ids' that will be sent to specific buckets (thanks @masco-enovance for your code). @hunchback is working on this code/.

Reply:
This is discussed in the section "Multiple pipelines".

Reply:
I would not add the multiple pipelines here, as in the context of secadvisor this is not needed. We can consider when we document the AWS flow logs, depending on how we decide to realize multi-tenancy.